spec/shared_context.rb

```diff
@@ -121,3 +116,30 @@ def integration_context_debugging
     end
   end
 end
+
+def system_context
```

Comment:
Do not remove them, just move to another method, or leave it there.
The problem is you start this (semantic) block in the method:
```ruby
  # Create client objects:
  $config ||= OneviewSDK::Config.load(@config_path)
  $config_synergy ||= OneviewSDK::Config.load(@config_path_synergy)
end
```

And then finishes it in the `before(:each)`:
```ruby
    system_context # You started the 'create client objects' inside this method 
    $client_120 ||= OneviewSDK::Client.new($config.merge(api_version: 120))
    $client     ||= OneviewSDK::Client.new($config.merge(api_version: 200))

    allow_any_instance_of(OneviewSDK::Client).to receive(:appliance_api_version).and_call_original
    allow_any_instance_of(OneviewSDK::Client).to receive(:login).and_call_original
```

My point is that you're splitting in the middle of a semantic block that should not be splitted, these lines should be together, they make sense to be together.

So or you create a method `load_client_configuration` and put all of these lines inside the method, or you leave it all on the before(:each).

Right now, it just seems and arbitrary split in the code to make rubocop not complain, that is breaking the logical flow.